dspace-api/src/test/java/org/dspace/builder/AbstractBuilder.java

```diff
@@ -178,7 +178,12 @@ public static void init() {
         systemWideAlertService = DSpaceServicesFactory.getInstance().getServiceManager()
                                                       .getServicesByType(SystemWideAlertService.class).get(0);
         try {
-            submissionConfigService = SubmissionServiceFactory.getInstance().getSubmissionConfigService();
+            SubmissionServiceFactory submissionServiceFactory = SubmissionServiceFactory.getInstance();
+            if (submissionServiceFactory != null) {
+                submissionConfigService = submissionServiceFactory.getSubmissionConfigService();
+            } else {
+                log.error("ERROR: Error initializing submissionServiceFactory");
```

Comment:
I find the changes to this file to be odd.  I don't understand why this small PR suddenly requires extra error handling here.  That almost implies to me that we are missing something & we have an unintended side effect to how the SubmissionConfigService is loading.